lib/model/CMetricModel.cc

```diff
@@ -577,6 +579,8 @@ void CMetricModel::fill(model_t::EFeature feature,
     const TSize2Vec1Vec& correlates{model->correlates()};
     const TTimeVec& firstBucketTimes{this->firstBucketTimes()};
     core_t::TTime bucketLength{gatherer.bucketLength()};
+    bool skipAnomalyModelUpdate = this->shouldIgnoreSample(feature, pid, model_t::INDIVIDUAL_ANALYSIS_ATTRIBUTE_ID,
+                                                           model_t::sampleTime(feature, bucketTime, bucketLength));
```

Comment:
I wasn't quite sure as to what the right `time` parameter is. Could you confirm @tveasey ?

Reply:
Yes I think this is correct. The difference is small, but this means that for metric valued conditions we get the prediction at exactly the time of the the data point was received.